src/sidebar/components/test/MarkdownEditor-test.js

```diff
@@ -373,6 +384,103 @@ describe('MarkdownEditor', () => {
     assert.deepEqual(wrapper.find('MarkdownView').prop('style'), textStyle);
   });
 
+  context('when @mentions are enabled', () => {
+    function typeInTextarea(wrapper, text = '@johndoe', key = undefined) {
```

Comment:
I think tests would be easier to follow if there wasn't a default value for `text`.